pkg/config/config.go

```diff
@@ -358,6 +358,9 @@ func GetYAML(data interface{}) (string, error) {
 
 // OpenPiperFile provides functionality to retrieve configuration via file or http
 func OpenPiperFile(name string, accessTokens map[string]string) (io.ReadCloser, error) {
+	if len(name) == 0 {
```

Comment:
This change was NOT compatible!!! The NO_FILE_FOUND error got lost. This broke the unit tests on a pull requests and I think also others check if the file was not found which is not possible anymore. How to continue with that? @OliverNocon